Assets/MRTK/Core/Inspectors/Utilities/MixedRealityInspectorUtility.cs

```diff
@@ -104,7 +106,17 @@ public static void AddMixedRealityToolkitToScene(MixedRealityToolkitConfiguratio
                 {
                     newInstance.ActiveProfile = configProfile;
                 }
-                EditorSceneManager.MarkSceneDirty(EditorSceneManager.GetActiveScene());
+
+                if(!newInstance.ActiveProfile.ExperienceSettingsProfile.IsNull())
+                {
+                    // Add a MixedRealitySceneContent object to a scene. Children of this object will scale appropriately dependent on MR platform
+                    MixedRealitySceneContent contentAdjuster = new GameObject("MixedRealitySceneContent").AddComponent<MixedRealitySceneContent>();
```

Comment:
Do we want to update all our example scenes to use this pattern?

Reply:
Yeah, will work on migrating them in a followup PR